frontend/src/routes.js

```diff
@@ -2,4 +2,5 @@ import Main from './lib/Main.svelte'
 
 export const routes = {
   '/': Main,
```

Comment:
I'd say you should create another component just for `/` so you don't have to have strange conditions inside `Main`.